spark-testing/docker_utils.py

```diff
@@ -0,0 +1,43 @@
+import sdk_cmd
+import sdk_networks
+
+
+def docker_exec(task, cmd):
```

Comment:
Currently, only Spark is using Docker and we have a few tests we need to verify both UCR and Docker behaviour (networking mostly). `docker_utils` don't seem to be useful in SDK testing library because SDK doesn't use Docker.